src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs

```diff
@@ -159,21 +159,9 @@ string GenerateCommandLineCommands (string ManifestFile, string currentAbi, stri
 			if (PackageName != null)
 				cmd.AppendSwitchIfNotNull ("--custom-package ", PackageName.ToLowerInvariant ());
 			
-			if (AdditionalResourceDirectories != null) {
-				foreach (var dir in AdditionalResourceDirectories) {
-					if (!Directory.Exists (dir.ItemSpec))
-						continue;
-					var flatArchive = Path.Combine (dir.ItemSpec, "..", "compiled.flata");
-					if (!File.Exists (flatArchive))
-						continue;
-					cmd.AppendSwitchIfNotNull ("-R ", flatArchive);
-				}
-			}
-			if (AdditionalAndroidResourcePaths != null) {
-				foreach (var dir in AdditionalAndroidResourcePaths) {
-					if (!Directory.Exists (dir.ItemSpec))
-						continue;
-					var flatArchive = Path.Combine (dir.ItemSpec, "compiled.flata");
```

Comment:
I am confused by `AdditionalAndroidResourcePaths`, can it be removed now?

Is this code using the correct array? https://github.com/xamarin/xamarin-android/blob/78de5e19f42201b25351dbc88573ca150ffb0cdf/src/Xamarin.Android.Build.Tasks/Tasks/Aapt2Link.cs#L268-L271

Reply:
We still need `AdditionalAndroidResourcePaths` as it allows us to find the Manifest files. The new property `AdditionalResourceArchives` unfortunately doesn't point to this location bit to the new `flata` folder

Reply:
I wonder if we can just add some meta data..